cms/djangoapps/contentstore/management/commands/export.py

```diff
@@ -14,16 +14,19 @@
 
 class Command(BaseCommand):
     """
-    Export the specified data directory into the default ModuleStore
+    Export the specified course into a directory.  Output will need to be tar zxcf'ed.
     """
-    help = 'Export the specified data directory into the default ModuleStore'
+    help = 'Export the specified course into a directory'
 
     def add_arguments(self, parser):
         parser.add_argument('course_id')
         parser.add_argument('output_path')
 
     def handle(self, *args, **options):
-        """Execute the command"""
+        """
```

Comment:
Used your wording, but rewrote to reflect the sad reality that it just throws it into the folder you give it.